cs-office-365/src/main/java/io/cloudslang/content/office365/utils/HttpUtils.java

```diff
@@ -160,4 +163,22 @@ private static String getLoginType(@NotNull final String userPrincipalName,
         return userId;
     }
 
+    @NotNull
+    public static Map<String, String> getOperationResults(Map<String, String> result) {
+        Map<String, String> results;
+        final String statusCode = result.get(STATUS_CODE);
+        if (Integer.parseInt(statusCode) >= 200 && Integer.parseInt(statusCode) < 300) {
```

Comment:
This if's execution branches contains duplicate code.

`results.put(STATUS_CODE, statusCode);
             results.put(DOCUMENT, result.get(RETURN_RESULT));`